stor/tests/shared_obs.py

```diff
@@ -207,3 +207,21 @@ def test_seek_behavior_and_writes(self, mock_write_object):
             obj.seek(0)
         mock_write_object.assert_called_with(self.normal_path, b'happy-go_lucky')
         self.assertEquals(len(mock_write_object.call_args_list), 1)
+
+    def test_readable_writable_seekable(self):
+        pth = self.normal_path
+        read_obj = pth.open(mode='r')
+        self.assertTrue(hasattr(read_obj, 'readable'))
```

Comment:
Nit: Split each on into a separate test.

Reply:
I disagree with this nit - it's great to have 'em all together, I think really nbd

Reply:
Unit tests should test a unit. These functions are separate units, therefore separate tests. of course they can all be next to each other.

Reply:
I'm fine with it either way.